### PR TITLE
Order vulnerabilities by severity #35

### PR DESCRIPTION
**Pull request by YoussefEgla**

### CODE REVIEW

**Comment by YoussefEgla** [comment]
Testing suite failed due to undefined value in `extension-api.js`

```
```

**Comment by YoussefEgla** [comment]
Test suite fails due to undefined value passed instead of a string in `extension-api.js` in `@k8slens` package

```
> @aquasecurity/starboard-lens-extension@0.0.1-alpha.7 test U:\starboard-lens-extension
> jest

 FAIL  src/components/vulnerabilities-list.test.tsx
  ● Test suite failed to run

    TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined

      at new Ee.constructor (node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:883:1651768)
      at new <anonymous> (node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:883:1653852)
      at Module.<anonymous> (node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:883:1651338)
      at r (node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:1:247)
      at node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:1:1046
      at Object.<anonymous> (node_modules/@k8slens/extensions/dist/src/extensions/extension-api.js:1:1058)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        16.426 s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

File: src/vulnerability-report.ts [hunk 1]
```diff
@@ -9,7 +19,15 @@ export class Vulnerability {
     severity: string;
     links?: string[];
 
-    constructor(params: Vulnerability) {
+    //
```
Comment (danielpacak) [inline src/vulnerability-report.ts:22]: You can leave such comment as PR comment, but it's not useful as part of the source code.

**Review by danielpacak — CHANGES_REQUESTED** [review]
@YoussefEgla I think that for ordering a list of Vulnerability objects by severity we should strive for a simple solution that works. And then maybe in subsequent PRs we could introduce testing frameworks and focus on refactoring. Also there's no point in adding lots of new dependencies and tests that are failing.

**Comment by YoussefEgla** [comment]
thanks so much for this comprehensive review You are right the built in sort method will do in order, i didn't notice

**Comment by danielpacak** [comment]
These changes are huge, break the build, have failing tests, and do not sort vulnerabilities by severity. Closing in favour of smaller PRs that actually work.

**Pull request closed without merging**